certified-connectors/Maximizer/apiProperties.json

```diff
@@ -0,0 +1,518 @@
+                "description": "Sign in with your Azure Active Directory credentials",
+                "tooltip": "Provide  Azure Active Directory credentials",
+                "constraints": {
+                  "required": "true",
+                  "hidden": "false"
+                }
+              }
+            },
+            "token:customHost": {
+              "type": "string",
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "uk1.maximizercrmlive.com"
+                    }
+                  ]
+                },
+                "displayName": "Host",
+                "description": "Select default to continue",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customBasePath": {
+              "type": "string",
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "/intMSFlows"
+                    }
+                  ]
+                },
+                "displayName": "Integration path",
+                "description": "Select default to continue",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customClientId": {
+              "type": "string",
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "{{OAuth2UKcustomClientId}}"
+                    }
+                  ]
+                },
+                "description": "Select default to continue",
+                "displayName": "Client Id",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customClientSecret": {
+              "type": "string",
+
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "{{OAuth2UKcustomClientSecret}}"
+                    }
+                  ]
+                },
+                "description": "Select default to continue",
+                "displayName": "Client Secret",
+                "tooltip": "Save your notes about this connection."
+              }
+            }
+          }
+        }
+      ]
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@MaximizerIntegration this is our standard icon brand color. Unfortunately, partners are not allowed to use it, may I ask you please to choose a different color. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector). Thank you.